localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```
@@ -224,4 +224,14 @@
  innovation_draenei_tactics_desc:0 "The Peacekeepers defend us from outside threats with all their might. When needed, these great fighters will wage our wars with unwavering spirit. The Rangari, on the other hand, ensure with their marksmanship that our enemies have no chance to even reach us."
 
  innovation_titan_tactics:0 "Pantheon's Chosen"
- innovation_titan_tactics_desc:0 "Few can claim the honor of being chosen by the Titans to be imbued with immense power, but these soldiers are among them. Capable of wielding powers comparable to those of demigods, these chosen soldiers are truly a force to be reckoned with."
\ No newline at end of file
+ innovation_titan_tactics_desc:0 "Few can claim the honor of being chosen by the Titans to be imbued with immense power, but these soldiers are among them. Capable of wielding powers comparable to those of demigods, these chosen soldiers are truly a force to be reckoned with."
+ 
+ ##############################
+ # Adapted Vanilla Traditions #
+ ##############################
+ 
+ culture_parameter_unlock_maa_mountaineer:0 "Can recruit [GetMaA('mountaineer').GetName] as [men_at_arms|E]"
+
+ innovation_forest_wardens:0 "Forest Wardens"
+ innovation_forest_wardens_desc:0 "This culture has made its home in the uncharted depths of ancient forests and fiercely guards the pristine sanctity of their ancestral homes from unwelcome trespassers."
```

I would've gone here without something like "this culture"...
Maybe something like this?
```suggestion
 innovation_forest_wardens_desc:0 "Living in the uncharted depths of ancient forests, the pristine sanctity of our ancestral homes is fundamental to our wardens. Any unwelcome trespasser might realise that before setting eyes on them."
```